testsuite/cluster-test/src/cluster_builder.rs

```diff
@@ -137,7 +138,10 @@ impl ClusterBuilder {
                         let vault_config = VaultConfig {};
                         self.cluster_swarm.spawn_new_instance(
                             InstanceConfig {
-                                validator_group: i,
+                                validator_group: ValidatorGroup {
```

Comment:
Can you make `ValidatorGroup::new_for_index(u64)` and call it here and in other similar places?